apps/pg_subscriber/lib/pg_subscriber/tuple_data.ex

```diff
@@ -1,29 +1,41 @@
 defmodule PgSubscriber.TupleData do
-
   alias __MODULE__
   alias PgSubscriber.Column
 
   @type t :: %__MODULE__{
-    num_of_cols: integer,
-    columns: [Column.t()],
-  }
+          num_of_cols: integer,
+          columns: [Column.t()]
+        }
```

Comment:
Isn't this indentation weird? Do you like it more?

Reply:
Honestly, I don't really care about it. I am using official elixir formatter upon save, thats why it changed it.

I think that we should agree on the formatting strategy because otherwise we will have a lot of random changes that will clatter the PR diff (which I hate). I vote for using official formatter as it makes it easier for me to code (I don't need to care about proper spacing, just save it and it gets pretty :smile: ).

But I am adaptable. So if you are hardly against it we can think of something better.

Reply:
I am not hardly against it. I would just liked it better how it was before and did not understand why you changed it to this one (if you disliked it before or...).
I agree, we can use official formatting style.